_assets/stylesheets/_utilities.scss

```diff
@@ -62,6 +49,14 @@
   justify-content: center;
 }
 
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
```

Comment:
duplicate style, please fix

Reply:
Good catch Eagle Eyes. I created the following defect for this... 

https://rally1.rallydev.com/#/38108142417d/detail/defect/400175318692?fdp=true